src/state/nftMarket/hooks.ts

```diff
@@ -14,17 +13,17 @@ export const useFetchCollections = () => {
   const dispatch = useAppDispatch()
   useEffect(() => {
     dispatch(fetchCollections())
-    dispatch(fetchNftsFromCollections('0x60935f36e4631f73f0f407e68642144e07ac7f5e'))
+    dispatch(fetchNftsFromCollections(pancakeBunniesAddress))
```

Comment:
Should this maybe be populated via the collections object in Redux (that comes from the API) rather than a constant?

Reply:
Hmmn I realise it's maybe overkill when we only have one collection and we're trying to move quickly - but I think modifying the `fetchNftsFromCollections` to accept multiple addresses would allow us to do all of this dynamically?

Reply:
The only reason for the constant was because we had it hard-coded everywhere. Just to avoid lowercase and checksummed differences I put it into a temporary variable.